### PR TITLE
{toolchain}[dummy] intel/2016.02-GCC-4.9

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

~~requires #2524 (for `binutils-2.26.eb`)~~

### CODE REVIEW

**Comment by fgeorgatos** [comment]
@boegel: visual review ok here, thanks!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2653.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e791ce96bb1459062d1f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2111.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b14ee61ee1a8c47d0035 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2699.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/57228543b09fdf450a76 for a full test report.

**Comment by boegel** [comment]
reworked to stick to binutils 2.25 for this one

I'd like to get this in for EasyBuild v2.7.0...

Thoughts @wpoely86?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2148.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5050f82160ce9ace163d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2148.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7fde6f815e943558c9b9 for a full test report.

File: easybuild/easyconfigs/i/icc/icc-2016.2.181-GCC-4.9.3-2.25.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# list of regex for components to install
+# full list of components can be obtained from pset/mediaconfig.xml in unpacked sources
+# cfr. https://software.intel.com/en-us/articles/intel-composer-xe-2015-silent-installation-guide
+components = ['intel-comp', 'intel-ccomp', 'intel-icc', 'intel-openmp', 'intel-ipsc?_']
+
+dontcreateinstalldir = 'True'
+
+# license file
+import os
```
Comment (wpoely86) [inline easybuild/easyconfigs/i/icc/icc-2016.2.181-GCC-4.9.3-2.25.eb]: I thought we didn't need this anymore?

Comment (boegel) [inline easybuild/easyconfigs/i/icc/icc-2016.2.181-GCC-4.9.3-2.25.eb]: good point, I'll get rid of it

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 9 (9 easyconfigs in this PR)
node2034.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9e57e4a4e47f2f00ca15 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 9 (9 easyconfigs in this PR)
node2626.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/150e7bf6baa5dac9c916 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2653.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9bc668db847c0375648b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2111.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4db5396b6a7c416783a0 for a full test report.

**Comment by boegel** [comment]
Going in, thanks for the feedback @wpoely86, @ocaisa and @fgeorgatos!

**Pull request merged**